lib/react_on_rails/server_rendering_js_code.rb

```diff
@@ -40,13 +40,17 @@ def render(props_string, rails_context, redux_stores, react_component_name, rend
           var railsContext = #{rails_context};
         #{redux_stores}
           var props = #{props_string};
-          return ReactOnRails.serverRenderReactComponent({
-            name: '#{react_component_name}',
-            domNodeId: '#{render_options.dom_id}',
-            props: props,
-            trace: #{render_options.trace},
-            railsContext: railsContext
-          });
+          try {
+            return ReactOnRails.serverRenderReactComponent({
+              name: '#{react_component_name}',
+              domNodeId: '#{render_options.dom_id}',
+              props: props,
+              trace: #{render_options.trace},
+              railsContext: railsContext
+            });
+          } finally {
+            console.history = [];
```

Comment:
alternatively, we could move this into `serverRenderReactComponent`. I'll let you all decide

Reply:
This is a good solution.

Reply:
BTW, @Judahmeek @ahangarha @KhaledEmaraDev This line would be a great place to add a configurable cleanup code for SSR. For example, in the past, I saw weird side effects from MobX on SSR.

Reply:
@wyattades I see your point that we could move `console.history = [];` to here: https://github.com/shakacode/react_on_rails/blob/master/node_package/src/serverRenderReactComponent.ts#L12

One can argue that it makes sense to keep the generated JS as small as possible, so I like that solution.

@Judahmeek @alexeyr-ci what do you guys think?

I vote we:

1. rename the [current function `serverRenderReactComponent`](https://github.com/shakacode/react_on_rails/blob/master/node_package/src/serverRenderReactComponent.ts#L12) to something like `serverRenderReactComponentInternal` and not export it.
2. Create `serverRenderReactComponent` in the same file with the try/finally block.

Reply:
Easier to read.  Function is already too long.